maven-model-builder/src/main/java/org/apache/maven/model/interpolation/AbstractStringBasedModelInterpolator.java

```diff
@@ -86,10 +80,15 @@
     private final UrlNormalizer urlNormalizer;
 
     @Inject
```

Comment:
This `@Inject` is obsolete now. Move the new field up to the other private finals.